fitnessllm_dataplatform/batch_handler.py

```
@@ -27,13 +32,56 @@
+        """
+        user_ref = self.get_specific_user(user_id)
+
+        if data_source == FitnessLLMDataSource.STRAVA.value:
+            strava_refresh_token = (
+                user_ref.collection("stream")
+                .document(data_source.lower())
+                .get()
+                .to_dict()
+                .get("refreshToken")
+            )
+            assert strava_refresh_token
```

Ruff (0.8.2)

Use of `assert` detected

(S101)